### PR TITLE
Fix/resolve capability error

### PR DESCRIPTION
**Pull request by sujine2**
- Add query sender field
  - sender: Relayer address
  - querySender: Address of the user who requested the query

### CODE REVIEW

File: proto/ibc/applications/ibc_query/v1/query.proto [hunk 1]
```diff
@@ -42,4 +42,5 @@ message QueryCrossChainQueryResultResponse {
   string id = 1;
   QueryResult result = 2;
   bytes data = 3;
+  string sender = 4;
```
Comment (Woojinger) [inline proto/ibc/applications/ibc_query/v1/query.proto]: Why should the query requester receive sender field?

File: proto/ibc/applications/ibc_query/v1/tx.proto [hunk 2]
```diff
@@ -54,8 +54,9 @@ message MsgSubmitCrossChainQueryResult {
   QueryResult result = 3;
   bytes data = 4;
   string sender = 5;
+  string query_sender = 6;
```
Comment (Woojinger) [inline proto/ibc/applications/ibc_query/v1/tx.proto:57]: 1. Does sender field mean relayer and query_sender mean query requester?

2. query_sender -> querySender

**Review by Woojinger — COMMENTED** [review]
Please check my comment :)
And will there be new pr related to this pr in the relayer repository?

File: proto/ibc/applications/ibc_query/v1/query.proto (hunk 1 above)
Comment (sujine2) [reply proto/ibc/applications/ibc_query/v1/query.proto]: I deleted this field, but I didn't add it to the commit.,Thank you for check

File: proto/ibc/applications/ibc_query/v1/tx.proto (hunk 2 above)
Comment (sujine2) [reply proto/ibc/applications/ibc_query/v1/tx.proto:57]: 1. yes

Comment (Woojinger) [reply proto/ibc/applications/ibc_query/v1/tx.proto:57]: 2: sorry my mistake..

**Review by Woojinger — APPROVED** [review]
I check it.
Thank you

**Pull request merged**